### PR TITLE
Thumbnail reordering and participant pane enhancements.

### PR DESCRIPTION
**Pull request by jallamsetty1**
Match the order in the participants pane with that of the order in the filmstrip. Also move the participants with raised hand to the top of the list.
Move enableThumbnailReordering flag to testing section.

### CODE REVIEW

**Review by damencho — APPROVED** [review]

**Review by damencho — DISMISSED** [review]

**Comment by jallamsetty1** [comment]
Jenkins please test this please

**Comment by damencho** [comment]
Seems it still fails with that when opening participants pane

File: react/features/base/participants/reducer.js [hunk 1]
```diff
@@ -238,15 +230,11 @@ ReducerRegistry.register('features/base/participants', (state = DEFAULT_STATE, a
...
+        // Insert the new participant and sort it alphabetically.
+        const remoteParticipants = Array.from(state.sortedRemoteParticipants.keys());
 
-        sortedRemoteParticipants.push([ id, displayName ]);
-        sortedRemoteParticipants.sort((a, b) => a[1].localeCompare(b[1]));
-
-        // The sort order of participants is preserved since Map remembers the original insertion order of the keys.
-        state.sortedRemoteParticipants = new Map(sortedRemoteParticipants);
+        remoteParticipants.push(id);
+        state.sortedRemoteParticipants = _sortParticipantsByDisplayname(state, remoteParticipants);
```
Comment (hristoterezov) [inline react/features/base/participants/reducer.js:237]: I find the new version less efficient than the previous one because we loop trough the remote participants one additional time. Is this correct? Why did you change it?

**Review by hristoterezov — APPROVED** [review]
